terraform/modules/neptune-cluster/main.tf

```diff
@@ -0,0 +1,91 @@
+  neptune_cluster_parameter_group_name = var.enable_serverless ? "default.neptune1.2" : module.cluster_parameters[0].name
+  preferred_backup_window              = var.preferred_backup_window
+  preferred_maintenance_window         = var.preferred_maintenance_window
+  port                                 = var.port
+  replication_source_identifier        = var.replication_source_identifier
+  skip_final_snapshot                  = var.skip_final_snapshot
+  snapshot_identifier                  = var.snapshot_identifier
+  storage_encrypted                    = true
+  vpc_security_group_ids               = var.vpc_security_group_ids
+
+  dynamic "serverless_v2_scaling_configuration" {
+    for_each = var.enable_serverless ? [1] : []
+
+    content {
+      max_capacity = var.max_capacity
+      min_capacity = var.min_capacity
+    }
+  }
+}
+
+module "cluster_parameters" {
+  count = var.enable_serverless ? 0 : 1
+  source = "git::https://github.com/CBIIT/datacommons-devops.git//terraform/modules/neptune-cluster-parameter-group?ref=Neptune"
+
+  app              = var.app
+  env              = var.env
+  program          = var.program
+  enable_audit_log = var.enable_cloudwatch_logs_exports == ["audit"] ? true : false
+}
+
+module "instance_parameters" {
+  source = "git::https://github.com/CBIIT/datacommons-devops.git//terraform/modules/neptune-instance-parameter-group?ref=Neptune"
+
+  app            = var.app
+  env            = var.env
+  program        = var.program
+  enable_caching = var.enable_serverless ? false : var.enable_caching
+  query_timeout  = var.query_timeout
+}
+
+module "neptune_instance" {
+  source = "git::https://github.com/CBIIT/datacommons-devops.git//terraform/modules/neptune-instance?ref=Neptune"
+
+  app                          = var.app
+  env                          = var.env
+  program                      = var.program
+  auto_minor_version_upgrade   = var.auto_minor_version_upgrade
+  cluster_identifier           = aws_neptune_cluster.this.cluster_identifier
+  engine_version               = var.engine_version
+  instance_class               = var.enable_serverless ? "db.serverless" : var.instance_class
+  neptune_subnet_group_name    = aws_neptune_subnet_group.this.name
+  neptune_parameter_group_name = var.enable_serverless ? "default.neptune1.2" : module.instance_parameters.name
+  port                         = var.port
+}
+
+resource "aws_kms_key" "this" {
+  deletion_window_in_days = 7
+  description             = "Enforces encryption at rest for the ${terraform.workspace}-tier neptune cluster"
+  key_usage               = "ENCRYPT_DECRYPT"
+}
+
+resource "aws_kms_alias" "this" {
+  name          = "alias/${var.resource_prefix}-neptune-key"
+  target_key_id = aws_kms_key.this.id
+}
+
+resource "aws_neptune_subnet_group" "this" {
+  name        = "${var.resource_prefix}-neptune-subnets"
+  description = "subnet group for the ${var.env}-tier neptune cluster"
```

Comment:
If you remove the var.env variable, then replace ${var.env} with ${terraform.workspace}

Reply:
I resolved this myself. Committed the changes.